packages/number-input/src/NumberInput/NumberInput.types.ts

```diff
@@ -18,17 +19,15 @@ export const Direction = {
 
 export type Direction = (typeof Direction)[keyof typeof Direction];
 
-export const State = {
-  Error: 'error',
-  None: 'none',
-} as const;
+export const State = FormFieldState;
 
 export type State = (typeof State)[keyof typeof State];
```

Comment:
Instead of doing this, could you import `type FormFieldState` and use that on line 102?

Reply:
I think I'll leave this as-is for now since it's part of an existing pattern (albeit a slightly more verbose one that may need another look in the future)